Bermuda/Bermuda/Bat.h

```diff
@@ -1,29 +1,29 @@
 #pragma once
-#include "NPC.h"
-#include "Evasive.h"
+#include "InteractableNPC.h"
 #include "MovableEntity.h"
 #include "DrawableEntity.h"
 #include "CollidableEntity.h"
+#include "Weapon.h"
```

Comment:
Deze include word niet gebruikt in de header. Denk dat je deze naar de .cpp kan verplaatsen. Daar lijk je hem wel te gebruiken.